lib/Target/SubstraitPB/Export.cpp

```diff
@@ -37,16 +37,21 @@ DECLARE_EXPORT_FUNC(ModuleOp, Plan)
 DECLARE_EXPORT_FUNC(PlanOp, Plan)
 
 FailureOr<std::unique_ptr<Plan>> exportOperation(ModuleOp op) {
-  if (!op->getAttrs().empty())
+  if (!op->getAttrs().empty()) {
+    op->emitOpError("has attributes.");
```

Comment:
Nit: no trailing punctuation with emit errors.

Reply:
(here and below)

Reply:
Fixed.